pkg/registrar/transitgateway.go

```diff
@@ -771,6 +775,36 @@ func (r *TransitGateway) removeFromPrefixList(ctx context.Context, awsCluster *c
 	return nil
 }
 
+func (r *TransitGateway) getTGWGAttachmentSubnetsOrDefault(ctx context.Context, awsCluster *capa.AWSCluster) ([]string, error) {
```

Comment:
nitpick: I miss a func comment explaining a bit what this function does, what's the default and so on.